data/kahet/kahet missions.txt

```diff
@@ -738,3 +738,1128 @@ event "pacified ka'sei"
+
+mission "Ka'het: Patir Mystery 8A"
+	name "Lab deployment"
+	description "After taking the ground lab (weighing <tons>) from <stopovers>, reach <system> while still cloaked in order to set it up."
+	source "Aventine"
+	stopover "Babiali"
+	destination "Builder Settlement"
+	passengers 5
+	cargo "research lab and satellite" 97
+	blocked `Dusk contacts you soon after you reach the spaceport. "<first>, I advise you that the next mission is going to require <capacity>."`
+	to offer
+		has "Ka'het: Patir Mystery 7: done"
+	on offer
+		conversation
+			`It doesn't take long for you to see Dusk; he's waiting for you under a canopy next to your landing pad, right at the edge of Aventine's starport. "We are almost ready to go," he says, "but I have run into a problem. My plan for our next mission was to have a repeat of the last expedition, but this time carrying a temporary laboratory to the Builder site. I was able to gather most of the instruments I was looking for in the local scientific facilities, but none of the mobile laboratories I am authorized to requisition by default were in good enough condition to fly."`
+			`	"Because of that, before we can start our mission, you are going to need to make a stopover at <stopovers> - I was able to convince the prefects on Viminal to reallocate one of the outposts that they had just delivered there to our mission, given it has higher priority status compared to the mining expeditions they are setting up."`
+			`	You help Dusk transport the chosen instruments to your ship; you also make sure to leave enough space for the facilities you're going to take from <stopovers>. As you're walking back toward the ship's controls, Dusk reminds you to bring as many ramscoops with you as possible - you're going to need them in order to complete the mission plan while cloaked.`
+				accept
+	on stopover
+		dialog
+			`Dusk's coordinates indicate the border of a large canyon near the equator of <stopovers>. Unlike most cracks on the surface, this one shows signs of geological activity: you only narrowly avoid a stream ejecting from a geyser in the canyon as you pass overhead. As you land, a row of Remnant structures are clearly visible running parallel to the cliff.`
+			`	The local crew assist in loading the modular laboratory split across three pallets; with them arrives a separate package, containing a surveillance satellite to be deployed near Patir to scan the Ka'sei from orbit.`
+	on abort
+		# Do nothing.
+	on fail
+		dialog
+			`As soon as you approach the <planet>, you see that the Ka'sei are following your ship. You won't be able to enter the settlement safely anymore.`
+
+	npc save
+		government "Test Dummy"
+		personality unconstrained heroic nemesis
+		system "Patir"
+		fleet
+			variant
+				"Cloak Check" 10
+
+mission "Ka'het: Patir Mystery 8B"
+	invisible
```

Comment:
This didn't offer because Babiali has no spaceport and this will only offer in a spaceport right now.
```suggestion
	landing
	invisible
```